libraries/Arduino_CAN/examples/CANReadFilter/CANReadFilter.ino

```diff
@@ -0,0 +1,51 @@
+/*
+  CANRead
+
+  Receive and read CAN Bus messages
+  
+  
```

Comment:
It would be interesting to document the expectation, i.e. which messages are expected to pass through the filter, and which ones are supposed to be filtered out?